### PR TITLE
removes proto files from celestia-core

### PR DESCRIPTION
**Pull request by SweeXordious**
## Description

Moves proto files to [celestiaorg/spec](https://github.com/celestiaorg/spec).

Closes: https://github.com/celestiaorg/celestia-core/issues/566

### CODE REVIEW

File: proto/tendermint/abci/types.pb.go [hunk 1]
```diff
@@ -0,0 +1,13988 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: tendermint/abci/types.proto
```
Comment (evan-forbes) [inline proto/tendermint/abci/types.pb.go]: let's figure out why this is breaking the e2e test before merging, I'm guessing it has something to do this file, as it's the only go code that's changed

Comment (SweeXordious) [reply proto/tendermint/abci/types.pb.go]: I'll be investigating it shortly. Thanks

**Comment by SweeXordious** [comment]
For the `Protobuf` CI, will we still need it ? Since it basically checks if the `proto` files are not deleted...
Maybe move it to `spec` repo ?

**Comment by evan-forbes** [comment]
> For the Protobuf CI, will we still need it ? Since it basically checks if the proto files are not deleted...
Maybe move it to spec repo ?

yeah good idea, we no longer need to check if the proto is breaking here. I also don't think that we have to check that the proto is breaking in the specs repo either, as most PRs will be changes to the proto files.

imho, I think we can remove that check

**Comment by SweeXordious** [comment]
Sounds good, thanks. Will remove it then look at e2e test

**Comment by evan-forbes** [comment]
Was thinking about this a bit more, and it _might_ be the case that the e2e test is failing because we could be using mismatched versions of the proto files and tendermint. I would think that more tests would fail if that's the case, but either way, we need to make sure that we use matching versions of each. 

This could mean ditching the master branch all together, and only use the latest stable version, or some ABCI++ verison.

**Comment by evan-forbes** [comment]
great find!

**Comment by SweeXordious** [comment]
Sure, will do. Thanks

**Review by evan-forbes — APPROVED** [review]
nice!

**Comment by SweeXordious** [comment]
Can you do it please? I don't have permissions

**Pull request merged**